version.json

```diff
@@ -1,6 +1,6 @@
 {
   "version": "15.3-preview",
-  "buildNumberOffset": "103",
+  "buildNumberOffset": "105",
```

Comment:
Would be good to just take the master version of this file. No reason to bump the version in master.